website/content/blog/2023-11-getting-started-as-a-tag-env-contributor.md

```diff
@@ -0,0 +1,124 @@
+
+7. [Tackle GitHub Issues:](https://github.com/cncf/tag-env-sustainability/issues) If you have a knack for addressing specific challenges or if you're seeking a shorter commitment, GitHub issues might be your calling. Browse through them and identify where you can contribute. If you feel lost and you don't know where to start, a good starting point is always an issue with the ["Help wanted" label](https://github.com/cncf/tag-env-sustainability/labels/help%20wanted).
+
+8. [Events & Meetups:](https://tag-env-sustainability.cncf.io/events/) If you lean more towards communications and networking, you're in for a treat! The TAG ENV meets in-person at major conferences like KubeCon+CloudNativeCon and Open Source Summit. In addition TAG ENV organizes global events like the [Cloud Native Sustainability Week](https://github.com/orgs/cncf/projects/10/views/11) where both a virtual mini conference and local [cloud native meetups](https://community.cncf.io) were organized by a larger, international community. If you have ideas or want to organize an event dedicated to the cloud native sustainability where you live, the TAG ENV community would be thrilled to help you out and support you in making it happen!
+
+Every individual, irrespective of their expertise or background, has something valuable to offer. If the topic of environmental sustainability resonates with you, the TAG ENV awaits your unique touch. Dive in, and together, let's make a difference!
+
+## Can YOU help shape the future of the TAG Environmental Sustainability?
+
+Yes, you can!
+
+TAG ENV is a community-driven organization, and every member's input is highly valued and appreciated. If you have an idea that you would like to propose to the TAG ENV community, the first step is to check if this idea already exists as a project or otherwise. You can check the [issues](https://github.com/cncf/tag-env-sustainability/issues) page for closed and open issues where you discuss your idea or reopen an old discussion.
+
+Scope your idea and refer to the first-time contributors quickstart from our [contributor guide](https://github.com/cncf/tag-env-sustainability/blob/main/CONTRIBUTING.md#first-time-contributors). It is advisable to refer to the [governance folder](https://github.com/cncf/tag-env-sustainability/tree/main/governance) to get a better understanding of different processes in the TAG.
+
+Your idea can be phrased in the form of a detailed proposal, a blog post, or even a presentation. It may be a message in Slack or a GitHub issue. You can write a full-fledged proposal from the beginning and once your idea is documented, you can  share it with the group in [#tag-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6) channel on [CNCF Slack](https://cloud-native.slack.com/archives/C03F270PDU6), where the group members actively discuss and provide feedback on new ideas.
```

Comment:
```suggestion
Your idea can be phrased in the form of a detailed proposal, a blog post, or even a presentation. It may be a message in Slack or a GitHub issue. You can write a full-fledged proposal from the beginning and once your idea is documented, you can  share it with the group in [#tag-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6) channel on [CNCF Slack](https://cloud-native.slack.com/archives/C03F270PDU6), where the TAG actively discusses new ideas.
```